package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -187,6 +195,7 @@ const FileUploadPreviewWithContext = <
                         24 - // 24 = close icon size
                         24, // 24 = internal padding
                     },
+                    I18nManager.isRTL ? { writingDirection: 'rtl' } : { writingDirection: 'ltr' },
```

Comment:
```suggestion
                     {writingDirection: I18nManager.isRTL ? 'rtl' : 'ltr'}
```

Reply:
This was mistakenly commited while I was switching branches...

Reply:
Also removed in latest commit